packages/lesswrong/components/messaging/ConversationPage.tsx

```diff
@@ -35,88 +30,22 @@ const styles = (theme: ThemeType): JssStyles => ({
 
 /**
  * Page for viewing a private messages conversation. Typically invoked from
- * ConversationWrapper, which takes care of the URL parsing.
+ * InboxWrapper, which takes care of the URL parsing.
  */
 const ConversationPage = ({ conversationId, currentUser, classes }: {
```

Comment:
I've pulled out most of the functionality in this page into the `ConversationContents` component, which is also used by `FriendlyInbox`